DSharpPlus/InlineMediaTool.cs

```diff
@@ -71,65 +71,63 @@ public MediaFormat GetFormat()
             return this.format;
         }
 
-        using (BinaryReader br = new(this.SourceStream, Utilities.UTF8, true))
+        using BinaryReader br = new(this.SourceStream, Utilities.UTF8, true);
+        ulong bgn64 = br.ReadUInt64();
+
+        if (bgn64 == PNG_MAGIC)
         {
-            ulong bgn64 = br.ReadUInt64();
+            return this.format = MediaFormat.Png;
+        }
 
-            if (bgn64 == PNG_MAGIC)
-            {
-                return this.format = MediaFormat.Png;
-            }
+        if ((bgn64 & MASK32_LESSER) == AVIF_MAGIC_PART_1 && br.ReadUInt32() == AVIF_MAGIC_PART_2)
```

Comment:
i cannot make sense of these changes, please elaborate or revert/ensure it is equivalent to master